task/08-objects-tasks.js

```diff
@@ -23,7 +23,13 @@
  *    console.log(r.getArea());   // => 200
  */
 function Rectangle(width, height) {
-    throw new Error('Not implemented');
+    this.width = width;
+    this.height = height;
+    Rectangle.prototype.getArea = function getArea() {
+        return width * height;
+    }
```

Comment:
this is an expression not a statement so you need to add a semicolon

https://medium.com/launch-school/javascript-expressions-and-statements-4d32ac9c0e74